### PR TITLE
datasheets for the four datasets from DBC D1G1TAL

### PR DESCRIPTION
**Pull request by peter-sk**

### CODE REVIEW

**Comment by KennethEnevoldsen** [comment]
Looks great @peter-sk - I can't see the data on UCloud is it possible that I have missed something?

**Comment by peter-sk** [comment]
You were too fast. Look again :-)

File: docs/datasheets/dbc-abstracts.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+
```
Comment (KennethEnevoldsen) [inline docs/datasheets/dbc-abstracts.md:1]: Seems like we are missing the frontmatter.

We are notably also missing license field in the datasheet. It should just describe what we can and can't do with the data if no license is specified.

**Review by KennethEnevoldsen — COMMENTED** [review]
Ahh thanks see them now.

File: docs/datasheets/dbc-abstracts.md (hunk 1 above)
Comment (peter-sk) [reply docs/datasheets/dbc-abstracts.md:1]: What is the frontmatter?

Comment (peter-sk) [reply docs/datasheets/dbc-abstracts.md:1]: Also, I am not sure how the datasheets should be formatted. I oriented myself according to daradio. But there seem to be vastly different datasheet styles used.

What is the preferred style? And how to put a license that says:

Data may be used to train on and model may be published openly. But data may not be shared with third parties.

Comment (KennethEnevoldsen) [reply docs/datasheets/dbc-abstracts.md:1]: The frontmatter is a yaml segment at the beginning of a markdown separated by `---`, e.g.

```
---
{yaml}
---
{markdown}
```
In terms of content daradio is quite good, except for the frontmatter + license stuff. For an example of the frontmatter see e.g. [this](https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/main/docs/datasheets/dagw-gutenberg.md)

the frontmatter is intended for the sheet to be machine readable.

Comment (peter-sk) [reply docs/datasheets/dbc-abstracts.md]: Done!

File: docs/datasheets/dbc-abstracts.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
+---
+pretty_name: dbc-abstracts
+language:
+  - da
+  - en
+  - "no"
+  - sv
+  - ty
```
Comment (KennethEnevoldsen) [inline docs/datasheets/dbc-abstracts.md:8]: ty?

**Review by KennethEnevoldsen — APPROVED** [review]
General looks fine, feel free to merge

File: docs/datasheets/dbc-abstracts.md (hunk 2 above)
Comment (peter-sk) [reply docs/datasheets/dbc-abstracts.md]: Oops. I de-danished the country code, relying on ISO-639-1 instead.

**Pull request merged**